### PR TITLE
IA-1584 fix a bug where dateAccessed is null after migration

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml [hunk 1]
```diff
@@ -38,4 +38,5 @@
     <include file="changesets/20191204_jupyter_start_script.xml" relativeToChangelogFile="true" />
     <include file="changesets/20200122_cluster_followup_table.xml" relativeToChangelogFile="true" />
     <include file="changesets/20200123_cluster_table_gce_refactor.xml" relativeToChangelogFile="true" />
+    <include file="changesets/20200123_populate_date_accessed.xml.xml" relativeToChangelogFile="true" />
```
Comment (rtitle) [inline http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml]: .xml.xml

File: http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_populate_date_accessed.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="populate_date-accessed">
+        <sql>UPDATE RUNTIME_CONFIG r set r.dateAccessed = NOW() where r.dateAccessed is NULL</sql>
```
Comment (rtitle) [inline http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_populate_date_accessed.xml:5]: Need to then set a not-null constraint on the column

File: http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml (hunk 1 above)
Comment (rtitle) [reply http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml]: Also can fix date.. `20200213`

Comment (Qi77Qi) [reply http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml]: multi tasking is hard :)

Comment (Qi77Qi) [reply http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml]: I left the date the same just so it's obvious that we ran the migration on the same date...

File: http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_populate_date_accessed.xml (hunk 2 above)
Comment (Qi77Qi) [reply http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_populate_date_accessed.xml:5]: updated

**Review by rtitle — APPROVED** [review]

**Pull request merged**